src/Trans.js

```diff
@@ -298,6 +298,7 @@ export function Trans({
     ...interpolationOverride,
     defaultValue,
     ns: namespaces,
+    ...tOptions?.interpolation,
```

Comment:
Would this need a fallback with the optional chaining or the `undefined` or `null` can be passed in safely?

Reply:
If it does, maybe `...tOptions?.interpolation ?? {} (or whatever works here)`

Reply:
Nope:

```
var x = {...null} // results in {}
var x = {...undefined} // results in {}
```

![image](https://user-images.githubusercontent.com/1500684/100663687-e8ea5a80-3313-11eb-9e2d-426238f246fa.png)

Reply:
Thanks to both of you for having a look and helping me out :hugs: :pray: :gift_heart:

Reply:
seems codeclimate has a problem with optional chaining...
alternatively, don't use the optional chaining operator and default tOptions to {}:
![image](https://user-images.githubusercontent.com/1086194/100709109-cf95ec80-33ad-11eb-88a6-bfbef7969f91.png)

Reply:
Can you also add a test?

Reply:
@adrai Yeah sure :100:, I would add a test where interpolation is passed to Trans, and assert the tOptions.interpolation's value :tada:.